spec/calculate_format_spec.rb

```diff
@@ -34,6 +34,24 @@
       ).to eq("spm")
     end
 
+    it "is NOT an MPM with null n_enum" do
+      htitem = double(:htitem, ht_bib_key: '12345', bib_fmt: 'BK', n_enum: nil)
```

Comment:
Rather than trying to get the htitem/cluster to do what we wanted, basically we say -- what if htitem behaves like this? Then `CalculateFormat` should behave the way we want.